web-app/src/app/services/search.service.ts

```diff
@@ -0,0 +1,26 @@
+import { ListKeyManager } from '@angular/cdk/a11y';
+import { Injectable } from '@angular/core';
+
+// TODO: Remove with backend implementation
+declare var require: any
+let data: any = require('./data.json');
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SearchService {
+
+  constructor() { }
+
+  search(searchValue: string) : [string, string][] {
```

Comment:
_Think_ this is O(n).
Wonder if there is a more efficient method.
Neverthess, backend should be able to handle search once implemented.